README.md

```diff
@@ -242,14 +245,14 @@ Now that we have an overview on how TLS and cipher suite work, we can dig into d
 
 
 ## On authentication
-All transactions in a blockchain network are digitally signed with the public key(s) of the issuer(s), hence each message is authenticated per se. TLS has a set of cipher suites that allow us to disable authentication. In this case, if for the parties involved a signed message suffice, they can ignore extra certificates and move on.
+All transactions on a blockchain network are digitally signed with the public key(s) of the issuer(s), so each message is authenticated *per se*. TLS has a set of cipher suites that allow us to disable authentication. In this case, if a signed message satisfies the parties involved, they can ignore extra certificates and move on.
```

Comment:
This isn't a grammar issue, but a technical one.

@vrde I think the transactions are signed with the _private_ key(s) of the issuer(s), no?